src/prefect/utilities/storage.py

```diff
@@ -113,5 +112,5 @@ def extract_flow_from_module(
         for var in dir(module):
             if isinstance(getattr(module, var), prefect.Flow):
                 return getattr(module, var)
-                
+
     raise ValueError("No flow found in file.")
```

Comment:
@jacksund this isn't a big deal but maybe we should update this error message to say `module` instead of `file`

Reply:
Better yet, the name of the module even: `f"No flow found in {module_str}"`

Reply:
I left it thinking this would portray the same idea, but I think you're right. I'll change it.

Reply:
Thanks for being so responsive!